rules/no-skip-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+module.exports = function (context) {
+	var ava = createAvaRule();
+	return ava.merge({
+		MemberExpression: function (node) {
+			if (ava.isTestFile && node.property.name === 'skip' && util.isCallFromTestObject(node)) {
```

Comment:
Hum, tried using `ava.currentTestNode === node` but it doesn't work here. They don't reference the same thing (`node` references the `MemberExpression`, while `currentTestNode` references the `CallExpression`, aka the parent of `node`).

I tried using `ava.isTestFile && ava.currentTestNode && ava.hasTestModifier('skip')` instead, but then I get two errors instead of one for `test.skip(...)` as it gets triggered for two of the nodes it passes through, and 4 for `test.skip.cb(...)`.
So I'm sticking to this implementation except if someone has a better idea.